eth/stagedsync/exec3.go

```diff
@@ -156,6 +157,8 @@ func ExecV3(ctx context.Context,
 	chainConfig, genesis := cfg.chainConfig, cfg.genesis
 	blocksFreezeCfg := cfg.blockReader.FreezingCfg()
 
+	agg.SetProduceMod(blocksFreezeCfg.ProduceE3)
```

Comment:
please move it to place where agg is creating. `backend.go`

Reply:
moved setting and checked all the functions for the correct produce setting